### PR TITLE
Fix explore.md

### PR DESCRIPTION
**Pull request by shiatsumat**
Fixed typos and minor errors in `explore.md`.

### CODE REVIEW

File: docs/explore.md [hunk 1]
```diff
@@ -26,16 +26,16 @@ This is an explicit non-goal of the project.
...
 This dynamic consensus that an object is no longer in use can cause 
 either latency issues or throughput.
 There is amazing research in doing this better
 but in Verona we want to take an alternative approach: ownership.
 
-Ownership has been used a in a few languages to support scalable memory management.
+Ownership has been used in a few languages to support scalable memory management.
 Rather than allowing multiple threads to access a single object, we ensure that at most one thread can access an object at a time.
-This is fundamental in languages like [Pony](https://www.ponylang.io) and [Rust](https://www.rust-lang.org).
+This is fundamental in languages like [Pony](www.ponylang.io) and [Rust](https://www.rust-lang.org).
```
Comment (mjp41) [inline docs/explore.md]: Looks like you edit is reverting #28.

File: docs/explore.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ There are two distinct aspect to system programming:
   - Can treat memory directly as bits and bytes
   - Little or no abstraction on the hardware
 
-To implement many low-level systems, for instance a memory manager, you need raw access.
+To implement many low-level systems (for instance, a memory manager), you need raw access.
```
Comment (davidchisnall) [inline docs/explore.md]: If you're going to correct the punctuation here, it's probably also a good idea to correct the vocabulary.  'For instance' is informal and should be 'for example' in formal writing.

Comment (shiatsumat) [reply docs/explore.md]: OK, I'll change 'for instance' to 'for example'.
Rather, I found 'many low-level systems' vague here; 'a low-level system of various kinds' may be clearer?

File: docs/explore.md (hunk 1 above)
Comment (shiatsumat) [reply docs/explore.md]: Oops, sorry and thank you for notifying this.

**Pull request merged**